builtin/common/after.lua

```diff
@@ -31,11 +31,16 @@ function core.after(after, func, ...)
 	assert(tonumber(after) and type(func) == "function",
 		"Invalid minetest.after invocation")
 	local expire = time + after
-	jobs[#jobs + 1] = {
+	local new_job = {
 		func = func,
 		expire = expire,
 		arg = {...},
-		mod_origin = core.get_last_run_mod()
+		mod_origin = core.get_last_run_mod(),
+		cancel = function(self)
```

Comment:
This job table is not exposed to mods, hence this cancel function cannot be used -> superfluous.